proto/deprecated.go

```diff
@@ -85,8 +87,27 @@ func UnmarshalJSONEnum(m map[string]int32, data []byte, enumName string) (int32,
 // Deprecated: Do not use.
 type InternalMessageInfo struct{}
 
-func (*InternalMessageInfo) DiscardUnknown(Message)                        { panic("not implemented") }
-func (*InternalMessageInfo) Marshal([]byte, Message, bool) ([]byte, error) { panic("not implemented") }
-func (*InternalMessageInfo) Merge(Message, Message)                        { panic("not implemented") }
-func (*InternalMessageInfo) Size(Message) int                              { panic("not implemented") }
-func (*InternalMessageInfo) Unmarshal(Message, []byte) error               { panic("not implemented") }
+// Deprecated: Do not use.
```

Comment:
I suggest adding more detailed info about the deprecation. Include such as:
- why this would be deprecated?
- when this would be removed from codebase?
- how to migrate if the user happens using this?

Reply:
Same as below.

Reply:
I'd prefer not. I can mention that it's an internal type, but users really should not be looking at these.